sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ReshuffleTest.java

```diff
@@ -274,4 +277,26 @@ public void testReshuffleWithTimestampsStreaming() {
 
     pipeline.run();
   }
+
+  @Test
+  @Category({ValidatesRunner.class, UsesTestStream.class})
```

Comment:
Is `TestStream` used in this test? otherwise we should remove the `UsesTestStream` annotation or the test won't be run for some runners.

Reply:
No, ValidatesRunner makes sense (thanks for fixing the other one), basically it says that all runner implementations should pass this test.